tests/apollo/test_skvbc_linearizability.py

```diff
@@ -40,8 +41,8 @@ def start_replica_cmd(builddir, replica_id):
             "-k", KEY_FILE_PREFIX,
             "-i", str(replica_id),
             "-s", statusTimerMilli,
-            "-v", viewChangeTimeoutMilli
-            ]
+            "-v", viewChangeTimeoutMilli,
+            "-p" if os.environ.get('BUILD_ROCKSDB_STORAGE') == "true" else ""]
```

Comment:
maybe don't check against a specific value here... Just verify if the variable is set at all, to avoid weird regressions in the future.

Reply:
on second thought, let's check against TRUE or ON (should be case insensitive)